examples/hbase-snapshot-migration-scripts/export-snapshot.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+# This script creates and exports hbase snapshots to another cluster
+# Run this script on HBase 1x cluster
```

Comment:
Any node or just the master node?

Reply:
master node. I'll add that to the comment